lib/src/lib_edit_address/models/country_model.dart

```diff
@@ -42,5 +42,5 @@ class CountryModel with EquatableMixin implements PickerItemModel {
   List<Object?> get props => [countryName, countryCode];
 
   @override
-  String? get itemDisplayName => countryName;
+  String get itemDisplayName => countryName;
```

Comment:
Please add information to the package including 
-pubspec.yank
- change log 
- documentation